ipn/ipnlocal/local.go

```diff
@@ -453,6 +453,24 @@ func (b *LocalBackend) updateStatus(sb *ipnstate.StatusBuilder, extraLocked func
 	}
 }
 
+// exitNodeIP returns the exit node IP from prefs.
+func (b *LocalBackend) exitNodeIP() netaddr.IP {
+	if !b.prefs.ExitNodeIP.IsZero() {
+		return b.prefs.ExitNodeIP
+	} else {
+		// convert exit node ID to IP
+		for _, bP := range b.netMap.Peers {
```

Comment:
what does `bP` stand for? How about just `p` for "peer"?

Reply:
backend Peer but I'll change to p